libs/server/Metrics/GarnetServerMonitor.cs

```diff
@@ -258,6 +258,7 @@ private async void MainMonitorTask(CancellationToken token)
                     var garnetServer = ((GarnetServerBase)server);
                     globalMetrics.total_connections_received = garnetServer.get_conn_recv();
                     globalMetrics.total_connections_disposed = garnetServer.get_conn_disp();
+                    globalMetrics.total_connections_active = garnetServer.get_conn_active();
```

Comment:
use `received - disposed` to compute this instead of `activeHandlers.Count` so that the three reported numbers are self-consistent.